src/lib/features/project/project-service.e2e.test.ts

```diff
@@ -9,7 +9,7 @@ import { RoleName } from '../../types/model';
 import { randomId } from '../../util/random-id';
 import EnvironmentService from '../project-environments/environment-service';
 import IncompatibleProjectError from '../../error/incompatible-project-error';
-import { ApiTokenService, EventService } from '../../services';
+import { type ApiTokenService, EventService } from '../../services';
```

Comment:
cool syntax. you don't see it often :)

Reply:
this is all biome's auto-fix black magic 🪄  but I'll take it 😄